en/orm/saving-data.rst

```diff
@@ -770,21 +774,20 @@ The ``save()`` method will return the modified entity on success, and ``false``
 on failure. You can disable rules and/or transactions using the
 ``$options`` argument for save::
 
-    // In a controller or table method.
+    // In a controller or table method
     $articles->save($article, ['checkRules' => false, 'atomic' => false]);
 
 Saving Associations
 -------------------
 
-When you are saving an entity, you can also elect to save some or all of the
-associated entities. By default all first level entities will be saved. For
-example saving an Article, will also automatically update any dirty entities
-that are directly related to articles table.
+When you are saving an entity, by default all associated entities,
+that are marekd dirty will be saved. You can however elect to save
+no associated entities by setting the ``associated`` option to ``false``.
```

Comment:
> I'm pretty sure only first-level associations are saved for free, and associations are not recursed into by default.

@markstory That's not what the docblock for `Table::save()` says:

>   * - associated: If true it will save all associated entities as they are found
     *   in the passed `$entity` whenever the property defined for the association
     *   is marked as dirty. Associated records are saved recursively unless told
     *   otherwise. If an array, it will be interpreted as the list of associations
     *   to be saved. It is possible to provide different options for saving on associated
     *   table objects using this key by making the custom options the array value.
     *   If false no associated records will be saved. (default: true)